Feature INCLUDE keyword (#224)
FPGA-Research-Manchester/FABulous: docs/source/fabric_definition.rst

   :emphasize-lines: 1,12

   TILE, LUT4AB      # define tile name            
   INCLUDE, ../include/Base.csv
   #direction  source_name  X-offset  Y-offset  destination_name  wires
   NORTH,      N1BEG,       0,        1,        N1END,            4
   EAST,       E2BEG,       1,        0,        N2END,            6
   JUMP,       J_BEG,       0,        0,        J_END,            12
   ...
   #         RTL code                     optional prefix
   BEL,      LUT4c_frame_config_OQ.vhdl,  LA_
   BEL,      LUT4c_frame_config_OQ.vhdl,  LB_
   ...
   MATRIX,   LUT4AB_switch_matrix.list               
   EndTILE  

> The ``INCLUDE`` Specify a path to another tile configuration, and the configuration in that file will be added. The 

Reviewer: ```suggestion
The ``INCLUDE`` keyword specifies a path to another tile configuration, and the configuration in that file will be added. The 
```
I think this reads a bit better :)

   MATRIX,   LUT4AB_switch_matrix.list               
   EndTILE  

> The path of the ``INCLUDE`` will relative to where the base file is. For example if the base file is located 

Reviewer: ```suggestion
The path of the ``INCLUDE`` will be relative to where the base file is. For example if the base file is located 
```

   # the same in compact form:
   N2BEG[0|0|0|0],[N2END3|E2END2|S2END1|LB_O]

> The ``INCLUDE`` keyword can also be used to include another list file to the current list file. For example:

Reviewer: ```suggestion
The ``INCLUDE`` keyword can also be used to include another list file in the current list file. For example:
```

.. code-block:: python

   INCLUDE, ../include/Base.list

   N2BEG0,N2END3 # cascade and twist wire index
   N2BEG0,E2END2 # turn from east to north
   N2BEG0,S2END1 # U-turn
   N2BEG0,LB_O   # route LUT B output north

and the target file ``../include/Base.list`` contains the following:

.. code-block:: python

   N1BEG0,N2END3 
   N1BEG0,E2END2 

The base file is equivalent to:
.. code-block:: python

   N1BEG0,N2END3 
   N1BEG0,E2END2
   N2BEG0,N2END3 # cascade and twist wire index
   N2BEG0,E2END2 # turn from east to north
   N2BEG0,S2END1 # U-turn
   N2BEG0,LB_O   # route LUT B output north

> The directory where the ``INCLUDE`` is relative to where the list file is located. For example if the file is located 

Reviewer: ```suggestion
The directory defined by the ``INCLUDE`` keyword is relative to where the list file is located. For example if the file is located 
```

I'm not entirely sure what the initial intention was, but this is what I understood it should mean, since the original sentence does not seem to be correct to me.
